client/src/components/UserProfile.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const UserProfile = ({ user }) => {
+  return (
+    <div>
+      <h1>This is a page built specifically for a user to see</h1>
+    </div>
```

Comment:
this is an incomplete feature. You arent actually displaying user information! Please work just a bit more on this